azure/postgresql/ServiceApprovalAcceleratorPostgreSQL.md

```diff
@@ -0,0 +1,417 @@
+                <li>Keep the newly created server (restored/replica) in an inaccessible state, because it’s unique identity hasn’t yet been given permissions to Key Vault.</li>
+                <li>On the restored/replica server, revalidate the customer-managed key in the data encryption settings. This ensures that the newly created server is given wrap and unwrap permissions to the key stored in Key Vault.</li>
+            </ol>
+    </td>
+    <td class="tg-0pky"><a href="https://docs.microsoft.com/en-us/azure/postgresql/concepts-data-encryption-postgresql">Encryption with Customer managed Keys</a></td>
+  </tr>
+  <tr>
+    <td class="tg-7btt" rowspan="3">Network Security </td>
+    <td class="tg-fymr">Endpoint Localisation</td>
+    <td class="tg-f8tv">
+        <p>
+            Private Link allows you to connect to various PaaS services in Azure via a private endpoint. Azure Private Link essentially brings Azure services inside your private Virtual
+            Network (VNet). The PaaS resources can be accessed using the private IP address just like any other resource in the VNet.
+        </p>
+    </td>
+    <td class="tg-0pky"><a href="https://docs.microsoft.com/en-us/azure/postgresql/concepts-data-access-and-security-private-link">Private Link for Azure Database for PostgreSQL-Single server</a></td>
+  </tr>
+  <tr>
+    <td class="tg-fymr">IP Firewall Rules</td>
+    <td class="tg-0pky">
+        <p> PostgreSQL service utilizes postgres built-in firewall mechanism using <code>ph_hba.conf</code> file. <br><br>
+            Connections to an Azure Database for PostgreSQL server are first routed through a regional gateway. The gateway has a publicly accessible IP, while the server
+            IP addresses are protected. The following diagram illustrates the traffic flow:</p>
+        <img src="media/net-security.png" width="700" height="300">
+        <p>
+            A newly created Azure Database for PostgreSQL server has a firewall that blocks all external connections. Though they reach the gateway, they are not allowed to
+            connect to the server. <br><br>
+            To begin using your server from another computer, you need to specify one or more server-level firewall rules to enable access to your server. Use the firewall rules
+            to specify which IP address ranges from the Internet to allow. Access to the Azure portal website itself is not impacted by the firewall rules. <br><br>
+            The example uses Access to Azure Services enabled or Public peering. This will be different for private link but gives some insight to how communications are proxied
+            to Azure PAAS services.<br><br>
+        </p>
+        <h4>Connecting from Azure</h4>
+        <p>
+            To allow applications from Azure to connect to your Azure Database for PostgreSQL server, Azure connections must be enabled. For example, to host an Azure Web Apps
```

Comment:
This is written strangely. It starts by saying that you need to enable azure connections to achieve (connecting from other apps in Azure). But the end of the paragraph is actually recommending to disable azure connections (and the subsequent point on data prevention also says best practice is to disable).  The consistency of what is being recommended isn't clear.